assets/js/events.js

```diff
@@ -243,13 +242,7 @@ function fillSidebarWithNodes (nodes, filter) {
       var href
 
       /* li.node */
-      li = $('<li>', {
-        'class': 'node'
-      })
-
-      if (element.hasOwnProperty('docs')) {
-        li.append($('<a/>').attr('class', 'toggle'))
-      }
+      li = '<li class="node">'
```

Comment:
maybe set `li_class = "node"` then down below do `li_class += " clicked"` and just then build the li?